factory/installScripts/webkit/install-webkit-version.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/node
+const { spawn } = require('child_process');
+
+const webkitVersion = process.argv.slice(2)[0]
+
+if (!webkitVersion) {
+  console.log('No Webkit version provided, skipping Webkit install')
+  return
+}
+
+// TODO: Verify that we don't have an arm64 version of webkit
+if (process.arch === 'arm64') {
+  console.log('Not downloading Webkit since we are on arm64')
+  return;
+}
+
+console.log('Installing Webkit version: ', webkitVersion)
```

Comment:
The scripts are ready to go, but they don't yet get installed because of [#25344](https://github.com/cypress-io/cypress/issues/25344) 

I figured i'd just check in the work instead of letting it rot on a branch.